.travis.yml

```diff
@@ -1,18 +1,29 @@
 language: python
 
+sudo: "required"
+
 python:
-  - "2.7"
-  - "3.5"
+    - "2.7"
+    - "3.5"
 
-sudo: "required"
+env:
+  global:
+    - DISPLAY=:99.0
+  matrix:
+    - PYTHON_2=2.7
+    - PYTHON_3=3.5
 
 before_install:
-  - "pip install virtualenv"
-  - "virtualenv venv"
-  - "source venv/bin/activate"
-
-install:
-  - "pip install -r requirements.txt "
+  - pip install mozdownload mozinstall
+  - mozdownload --type daily --branch mozilla-central --destination firefox.tar.bz2
+  - mozinstall firefox.tar.bz2
+  - sudo ln -sf $PWD/firefox/firefox /usr/local/bin/firefox
+  - pip install tox
```

Comment:
Let's instead use [tox-travis](https://pypi.python.org/pypi/tox-travis), which has better support for multiple python versions on Travis.